packages/mui-joy/src/Autocomplete/AutocompleteProps.ts

```diff
@@ -284,6 +284,15 @@ type AutocompleteOwnProps<
     variant?: OverridableStringUnion<VariantProp, AutocompletePropsVariantOverrides>;
   };
 
+  /**  
+   * AutocompleteProps signature:
+   *   export interface AutocompleteProps<
+   *     T, // the option structure
+   *     Multiple extends boolean | undefined,
+   *     DisableClearable extends boolean | undefined,
+   *     FreeSolo extends boolean | undefined,
+   *   >
+   */   
```

Comment:
Can you use `@template` as documented in this [link](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template).

Reply:
Thanks for your advice @siriwatknp 

I improved my notations by following the docs you mentioned, make sure everything is ok now, please